### ISSUE TITLE
bot commits lol

### ISSUE THREAD

**Issue by justinbalaguer** [gayanvoice/top-github-users#174]

**Comment by mauromorales** [comment]
indeed, all the top users are bots, or real users whose accounts have bot commits. At least there should be a way to report those users but I guess this is also a bot project LOL

**Comment by jayvdb** [comment]
I just looked through the top users in Jakarta, Indonesia, and most had an auto-commit bot.

One vector that can be used is to check each users username/username repo for an auto-commit bot, or look for excessive commits per day, and penalise or even omit the user.  Note I am advocating for this even though I have one of these on my profile account, but mine does about 10 commits per day https://github.com/jayvdb/jayvdb/commits/main/ , and I could easily reduce that if there was a sensible threshold.
